src/ert/run_models/everest_run_model.py

```diff
@@ -155,11 +154,18 @@ def __init__(
         simulation_callback: SimulationCallback | None,
         optimization_callback: OptimizerCallback | None,
     ):
-        everest_config = self._add_defaults(everest_config)
-
         Path(everest_config.log_dir).mkdir(parents=True, exist_ok=True)
         Path(everest_config.optimization_output_dir).mkdir(parents=True, exist_ok=True)
 
+        assert everest_config.environment is not None
+        logging.getLogger(EVEREST).info(
```

Comment:
Should this logging maybe also be moved to the environment config as it relates to the seed? Semi-related but I think the two logging statements could be merged into one, if it seems sensible to do in this PR

Reply:
I think the logging is appropriate here, since it is basically announcing that we are going to run optimization with e given seed. We could move it to the config, then it would be more like: logging that we modified the configuration to fill in a defaulted value. We generally don't seem to do that.

I will merge them.

Reply:
:heavy_plus_sign: